packages/core/build/tokens.ts

```diff
@@ -339,8 +339,8 @@ const typography = {
   },
   baseFontSize: token('125%'),
   baseFontSizePx: token(20),
-  fontFamily: token("'Clarity City', 'Avenir Next', 'Helvetica Neue', Arial, sans-serif"),
-  headerFontFamily: token("'Clarity City', 'Avenir Next', 'Helvetica Neue', Arial, sans-serif"),
+  fontFamily: token("'Clarity City', Metropolis, 'Avenir Next', 'Helvetica Neue', Arial, sans-serif"),
+  headerFontFamily: token("'Clarity City', Metropolis, 'Avenir Next', 'Helvetica Neue', Arial, sans-serif"),
```

Comment:
There are bugs with the glyph alignment in Metropolis which would cause this to break layouts. Has this change been visually tested?

Reply:
Yeah there are issues. I was thinking of adding since it's the fallback and breaks less than the other native font fallbacks. I had to do this in the recent hackathon since I couldn't get city to load so it was the next best fallback. I'll double check though to make sure I didn't miss anything.

Reply:
So testing Metropolis as a fallback is visually an improvement than the system fallback fonts. However thinking over this more, allowing it as a fallback might discourage teams from correcting the source issue of properly loading City in their apps. Also we don't know what version of Metropolis it might load, it may not be the latest that is in clr-ui which could cause more confusion. Going to close out this PR. Here are some of the screenshots 

### City
![city](https://user-images.githubusercontent.com/2021067/102516324-53a9ce80-4054-11eb-9d9b-72f0efef0260.png)

### Metropolis
![metropollis](https://user-images.githubusercontent.com/2021067/102516336-57d5ec00-4054-11eb-8627-1cebed14663f.png)  

### Avenir Next
![avenir-next](https://user-images.githubusercontent.com/2021067/102516458-789e4180-4054-11eb-9437-1cb0e3d0b293.png)

### Helvetica Neue
![helvetica-neue](https://user-images.githubusercontent.com/2021067/102516387-658b7180-4054-11eb-9cc6-bb4994eab751.png)

### Arial
![arial](https://user-images.githubusercontent.com/2021067/102516415-6e7c4300-4054-11eb-9ebf-209be680daa7.png)